string[] status
float32 dome_enc